agenda.md

```diff
@@ -0,0 +1,20 @@
+## Agenda topic best practices
+
+All TC39 agenda items should be posted on the agenda before the "agenda deadline", which is precisely 10 days before the start of the meeting. It's best to post agenda items as soon as you know you'll be presenting on the topic, even if you don't have all of the supporting materials yet.
+
+Before the agenda deadline, agenda items should be accompanied by supporting materials. Supporting materials includes slides, a link to the proposal repository, a link to spec text, etc.; essentially, anything you are planning to present to the committee, or that would be useful for delegates to review.
+
+Pull requests are encouraged for all agenda additions, especially for late additions after the deadline, so that members are notified of changes. All delegates have write access to the agenda, and should promptly land agenda additions so that they are maximally visible quickly.
```

Comment:
```suggestion
Pull requests are encouraged for all agenda modifications, especially for late additions after the deadline, so that members are notified. All delegates have write access to the agenda, and should promptly land agenda additions so that they are maximally visible quickly.
```